editors/mediawiki.el/Portfile

```diff
@@ -1,37 +1,39 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 PortSystem 1.0
+PortGroup github 1.0
 
-name                    mediawiki.el
-version                 2.2.4.1
+github.setup            hexmode mediawiki-el 2.3.1
+set myname              mediawiki.el
 categories              editors 
 license                 GPL-3
 maintainers             {easieste @easye} openmaintainer
 description             An Emacs mode for editing MediaWiki content
 long_description        ${description}
 homepage                http://www.emacswiki.org/emacs/MediaWikiMode
 platforms               darwin
-distname                mediawiki.el
-master_sites            https://launchpadlibrarian.net/127264410/
+distname                ${version}
```

Comment:
you shouldn't need to set the `distname`, the github PG should take care of that. Did you get an error when leaving this out?

Reply:
I didn't try leaving it out; I just changed it because leaving it as what it was previously set to caused incorrect behavior. I guess I can just try deleting that line entirely, though...

Reply:
ok fixed in c15660c